packages/cra-template/template/netlify.toml

```diff
@@ -0,0 +1,3 @@
+[build]
+  publish = "build"
+  command = "pnpm run build"
```

Comment:
this should probably have a newline at the end, so it doesn't have this symbol of a circle with line through it

Reply:
Added a line break in this commit
[e5e7376](https://github.com/upleveled/create-react-app/pull/2/commits/e5e737696539868eaea1f5fe67b7308177fbd5f4)